### PR TITLE
[REQUIRES #16571] TG sprite project: item/obj (part 1)

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
This PR ports applicable icons for the item/obj folder (no sub-folders). Anything they had in 3/4th perspective that didn't look like utter ass was ported over.

See also: https://github.com/ParadiseSS13/Paradise/pull/16571

This includes things such as:
- Chem machines
- Bodybags
- Lamps
- Scanner/copier
- Janitor cart
- Microwave
- Stools
- Filing Cabinets
- Glass reagent containers
- Stunbaton
- Telebaton
- Beds and Bedsheets
- Fax machine (from a skyrat PR, since TG don't use fax machines AFAIK)

New things:
Pews, corpo sofas, light AND dark metal chair variants.

## Why It's Good For The Game
Begun, the TG sprite port PRs have. This brings more items in line with our new style and perspective for sprites.

## Images of changes
There are quite a few things here, IDB will have an extensive list, I suggest you have a look there.

Here are the new chairs and fax machines though, since seeing how they smooth is hard in IDB
image

## Changelog
:cl:
tweak: Ported many TG icons for "objects"
add: Added Pews and Corpo Sofas, made with wood and metal respectively.
/:cl:

### CODE REVIEW

**Comment by S34NW** [comment]
Stuck an IDB ignore while i split this into smaller PRs as requested by AA

**Review by SabreML — COMMENTED** [review]
<details>
<summary><b>Review with lots of images:</b></summary>
<hr>

image
As far as I can tell, `bodybag_closed1` is never used anywhere in the code, so it should just be safe to delete it and avoid a duplicate sprite.
<hr>

image
Might just be me, but the little lips on the back of these look a bit odd.
<hr>

image
...Is there actually a change here? As far as I can tell the only difference is a couple of extra pixels on the sides.
<hr>

image
The hading looks a little funky on these, but I guess that could just be fixed in another PR.
<hr>

image
image
I have an irrational hatred of both of these sprites. The top half is fine, but the wierd knobbly bit at the end really just looks wierd.
The plastic cutlery all looks fine, so would it maybe be possible to just change it to that style for the handles?
image
<hr>

image
I also really dislike TG's beaker sprites. The colours look strange compared to every other glass thing in the game, and the sprites are absolutely massive:
image
image
<hr>

image image
(Also you probably want to move the 'filled' sprites into this PR too. Looks a bit odd otherwise.)
<hr>

image
That's... a very small beaker in the heater.
<hr>

image
Most of the paper bins around the station look a bit odd, presumably because the offsets aren't accurate anymore.
<hr>

image
:eyes:
<hr>

image
These lights look *really* out of place at the moment, but hopefully that's something that'll get resolved as more stuff is added.
<hr>

image
Not sure about the photocopier sprite. You can't really tell what it's meant to be without examining it.
<hr>

image
Same issue as the beaker overlays here with it not fitting properly. Won't be a problem if both PRs are merged at the same time though.
<hr>

</details>

File: code/modules/reagents/chemistry/machinery/chem_dispenser.dm [hunk 1]
```diff
@@ -16,7 +16,6 @@
 	var/recharge_counter = 0
 	var/hackedcheck = FALSE
 	var/obj/item/reagent_containers/beaker = null
-	var/image/icon_beaker = null //cached overlay
```
[the remark this answers is not in the document]
Comment (S34NW) [reply code/modules/reagents/chemistry/machinery/chem_dispenser.dm:19]: yeah, for now I've just got it animating when there is a beaker in it

**Review by SabreML — COMMENTED** [review]
Review 2!
<details>
<summary><b>More images:</b></summary>
<hr>

There seems to be some inconsistencies in height between the regular sofa parts and the corner pieces:
image
image

The 'Corporate' version seems to be smoothing fine so I assume it's not intentional.
image
<hr>

On that subject, I'm not entirely sure about the look of the corporate sofas.
I can't help seeing the shading on the front and back of the cushions as being badly laminated with air bubbles, really.
image
<hr>

Could you move the sprite for the folded tele-baton down a little bit so that it lines up with the extended version?
image
ezgif com-gif-maker
<hr>

image
This light should be green still I think, otherwise it looks like there's something wrong with it.
image
<hr>

The shading on the beaker/bottle lids seems a bit.. nonexistent? It might just be me, but they look pretty odd.
image
<hr>

https://user-images.githubusercontent.com/57483089/130366450-72f19ad0-b50a-454c-b4c7-29809555b63e.mp4
<hr>

The 'On' animation on the stunbatons looks a bit like it's in slow motion at the moment. Might be possible to speed it up a bit?
idXSbvlW6P
<hr>

Still don't like this sprite much but.. eh. It's fine.
image
<hr>
</details>

**Comment by PacifistDalek** [comment]
One of the other code bases resprited the caps for those beakers, I'll take a look and see if I can find it, and see if they look better

**Comment by S34NW** [comment]
> 
> 
> One of the other code bases resprited the caps for those beakers, I'll take a look and see if I can find it, and see if they look better

That would be ideal if they suit the TG aesthetic.

**Review by Kyep — APPROVED** [review]

**Review by variableundefined — APPROVED** [review]

**Pull request merged**